src/routes/usersRoute.js

```diff
@@ -0,0 +1,49 @@
+module.exports =  (app) => {
+
+    const userController = require('../controllers/userController.js')
+    userController.initializeDatabase()
+    
+
+    // Endopoints
+    
+    // POST /users  body: {"email": "<userEmail>", "password": "<userPassword>"} --> creates a user with <userEmail> and <userPassowrd> if it doesn't exists
+    app.post('/users', (req, res) => {
+        const userEmail = req.body.email
+        const userPassword = req.body.password
+        if(userEmail != undefined && userPassword != undefined)
+            (async () => {
+                if(await userController.registerUser(userEmail, userPassword))
+                    res.send("Se creo el usuario");
```

Comment:
Is there a significant difference between res.send and res.json in this case? Because in postman the only difference I see is the quotation marks around the message (res.send(...) -> User created  |  res.json(...) -> "User created")